### DISCUSSION TITLE
ruff . test

### DISCUSSION THREAD

**Discussion by pluto-tofu** [TheAlgorithms/Python#9721] [General]
I am getting these warnings while doing the ruff. test : 
warning: Selection of nursery rule `PLC1901` without the `--preview` flag is deprecated.
warning: Selection of nursery rule `PLC1901` without the `--preview` flag is deprecated.
warning: Selection of nursery rule `PLC1901` without the `--preview` flag is deprecated.
could anyone please guide me on how to fix these ?

**Comment by sozelfist** [comment]
May be this discussion will be helpful for you, [issue-7491](https://github.com/astral-sh/ruff/issues/7491). Just remove the `PLC1901` from `ignore` you configuration.
